addon/activity-notifier/service.js

```diff
@@ -139,6 +146,11 @@ export default Service.extend({
           `Draft by <b>${data.influencer_name}</b> in <b>${data.campaign_name}</b>
            <a href="${data.url}" target="_blank">Review</a>`
         );
+      case 'list_recommendation':
+        return messageWithoutAvatar(
+          `You have <b>${data.count}</b> recommendations for list <b>${data.list_name}</b>
```

Comment:
`You have <b>${data.count}</b> new recommendations for your list <b>${data.list_name}</b>`

Reply:
`You have <b>${data.count}</b> new recommendations for your <b>${data.list_name} list</b>`